src/components/attendees/EditAttendee.tsx

```diff
@@ -0,0 +1,129 @@
+import { ReactComponent as AdmitIcon } from '../../assets/Attendees/admit.svg';
+import { ReactComponent as CheckInIcon } from '../../assets/Attendees/checkin.svg';
+import { ReactComponent as DeleteIcon } from '../../assets/Attendees/delete.svg';
+
+import { Attendee } from '../../models/attendee';
+
+import { useState } from 'react';
+
+/***
+ * @return form to edit attendee details.
+ */
+export default function AttendeePopUp(props: { selectedAttendee: Attendee }) {
+	const labelStyle = 'text-[22px] max-w-[350px] font-medium text-left w-3/4';
+
+	const [attendee, setAttendee] = useState<Attendee>(props.selectedAttendee);
+	const inputStyle =
+		'w-3/4 max-w-[350px] p-2 bg-popup-input-bg border-solid border-popup-input-border border-[1px] rounded-lg shadow-sm bored-r-8 ';
+	const inputGroupStyle = 'w-full flex flex-col items-center justify-start';
+	const grayButtonStyle =
+		'bg-dark-action-color p-2 text-white w-3/4 max-w-[400px] rounded-lg font-bold text-lg border-2 border-solid border-dark-action-border flex items-center justify-center flex-row gap-3 hover:brightness-90';
+	const redButtonStyle =
+		'bg-red-action-color p-2 text-white w-3/4 max-w-[400px] rounded-lg font-bold text-lg border-2 border-solid border-red-action-border flex items-center justify-center flex-row gap-3 hover:brightness-90';
+
+	return (
+		<>
+			<span className="h-[1.5px] m-0 p-0 w-full bg-gray-200"></span>
```

Comment:
I think this is just a really complicated border.

Reply:
yea it's just the border LOL.